app/components/modals/PassphraseModalContent/Modal.js

```diff
@@ -0,0 +1,45 @@
+// @flow
+import React from "react";
+import KeyBlueButton from "KeyBlueButton";
+import { FormattedMessage as T } from "react-intl";
+import { PasswordInput } from "inputs";
+import "style/Modals.less";
+
+const Modal =({
```

Comment:
As I mentioned on another file, I'd move (or insert by reference) this inside PassPhraseModal (below the current title/content), as all PassphraseModals are supposed to ask for the password.